step-capstone/src/components/Sidebars/HandleClickedTimePoint.js

```diff
@@ -0,0 +1,75 @@
+/* Handling click into empty slot starts HERE */
+// Receives displayDate, returns the date string of displayDate
+const getDisplayDateString = (displayDate) => {
+  let year = displayDate.getFullYear().toString();
+  let month = (displayDate.getMonth() + 1 <10)? "0" + (displayDate.getMonth() + 1).toString() : displayDate.getMonth();
+  let date = (displayDate.getDate() < 10)? "0" + displayDate.getDate()   : displayDate.getDate();;
+  return year + "-" + month + "-" + date +   "T";
+};
```

Comment:
I feel like there should be a way to use standardized `Date` functionality to do this. Looks like the `toString` method might be what you want.